packages/webapp/components/layouts/ProfileLayout/NavBar.tsx

```diff
@@ -52,7 +52,7 @@ export default function NavBar({
       className={classNames(
         'sticky top-12 z-3 -mt-px flex justify-around bg-background-default tablet:top-14 tablet:justify-start',
         styles.nav,
-        isNewMobileLayout && 'tablet:!top-0',
+        isMobileLayout && 'tablet:!top-0',
```

Comment:
Then we can remove the `tablet:top-14` from above.
```suggestion
        'tablet:top-0',
```

Reply:
this ended up to be laptop:top-14 from what I understood, just adding comment for u to check as well 🙏

Reply:
Ah, yes, was a mistake with the suggestion, should be `laptop:top-14`.